CHANGELOG.md

```diff
@@ -0,0 +1 @@
+* Ensures `auth:export` results are fully flushed to the output file.
```

Comment:
Why is this bundled in with this PR?

Reply:
Sorry, this must be a leftover from an older PR I made.
I'm gonna clean this.